client/src/components/InputTextField/InputTextField.css

```diff
@@ -1,17 +1,16 @@
-.InputTextField--default input {
+.InputTextField__container-default {
```

Comment:
It looks like `default` is a modifier here. If so, it should be `InputTextField__container--default`

Reply:
Fixed per your comment